packages/hooks/src/useTotalAccountBalance.ts

```diff
@@ -0,0 +1,24 @@
+import { useStore } from '@deriv/stores';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+
+const useTotalAccountBalance = (accounts: DetailsOfEachMT5Loginid[]) => {
+    const { exchange_rates } = useStore();
+    const currency = accounts[0]?.currency;
+
+    const total_balance = accounts.reduce((total, account) => {
+        const base_rate = exchange_rates?.data?.rates?.[currency] || 1;
+        const rate = exchange_rates?.data?.rates?.[account.currency!] || 1;
```

Comment:
```suggestion
        const rate = exchange_rates.data?.rates?.[account.currency || 'USD'] || 1;
```
Not sure about `USD` 👀